strawberry/schema/schema_converter.py

```diff
@@ -309,7 +312,88 @@ def get_graphql_fields() -> Dict[str, GraphQLField]:
         return graphql_object_type
 
     def from_resolver(self, field: StrawberryField) -> Callable:
-        return field.get_wrapped_resolver()
+        def _get_arguments(
+            source: Any,
+            info: Any,
```

Comment:
Do `source` and `info` have known types now?

Reply:
info yes, source depends on the resolver, so I think any is good :)